assets/src/edit-story/components/inspector/inspectorProvider.js

```diff
@@ -6,26 +6,78 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { useState } from '@wordpress/element';
+import { useCallback, useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
+import { useAPI } from '../../app/api';
 import Context from './context';
 
 const DESIGN = 'design';
 const DOCUMENT = 'document';
 const PREPUBLISH = 'prepublish';
 
 function InspectorProvider( { children } ) {
+	const { actions: { getAllStatuses, getAllUsers } } = useAPI();
 	const [ tab, setTab ] = useState( DESIGN );
+	const [ users, setUsers ] = useState( [] );
+	const [ statuses, setStatuses ] = useState( [] );
+
+	const [ isUsersLoading, setIsUsersLoading ] = useState( false );
+	const [ isStatusesLoading, setIsStatusesLoading ] = useState( false );
+	const [ disabledStatuses, setDisabledStatuses ] = useState( [] );
+
+	const loadStatuses = useCallback( () => {
+		if ( ! isStatusesLoading && statuses.length === 0 ) {
+			setIsStatusesLoading( true );
+			getAllStatuses().then( ( data ) => {
+				data = Object.values( data );
+				// eslint-disable-next-line camelcase
+				data = data.filter( ( { show_in_list } ) => show_in_list );
```

Comment:
You could also just rename it inline to avoid the disable such as:

```js
				data = data.filter( ( { show_in_list:isShown } ) => isShown );
```

But yes, it's a bit silly.

Reply:
Works for me. Fixed in https://github.com/ampproject/amp-wp/pull/4122/commits/2ca09099dd8f3274206534faa0fec332ec7890c9